DashWallet/Sources/Models/CrowdNode/CrowdNode.swift

```diff
@@ -166,9 +166,7 @@ public final class CrowdNode {
     }
 
     private func topUpAccount(_ accountAddress: String, _ amount: UInt64) async throws -> DSTransaction {
-        let topUpTx = try await sendCoinsService.sendCoins(address: accountAddress,
-                                                           amount: amount)
-        return await txObserver.first(filters: SpendableTransaction(txHashData: topUpTx.txHashData))
+        try await sendCoinsService.sendCoins(address: accountAddress, amount: amount)
```

Comment:
It doesn't seem to be necessary to wait for the internal transaction to be "spendable". I did it by analog with Android, but in iOS outputs of internal transactions seem to be immediately available to spend.